File: C Language/LinearSearch.c
#include <stdio.h>

int main(void)
{

    int arr[100], i, n, x, flag = 0;

    printf("Enter the number of elements you want to enter:");
    scanf("%d", &n);

    printf("Enter the elements of the array:\n");
    for (int i = 0; i <= n; i++)
    {
        scanf("%d", &arr[i]);
    }

    printf("Enter the element to search: ");
    scanf("%d", &x);
    for (i = 0; i <= n; i++)
    {
        if (arr[i] == x)
        {
            printf("Element found at index %d\n", i);
            flag = 1;
            break;
        }
    }
    if (flag == 0)
    {
        printf("Element not found in the array\n");
    }
}

File: Java/PPS Java/linearSearch.java
import java.util.Scanner;

public class linearSearch {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.print("Enter the size of the array: ");
        int size = input.nextInt();
        int[] arr = new int[size];
        System.out.print("Enter the elements of the array: ");
        for (int i = 0; i < size; i++) {
            arr[i] = input.nextInt();
        }
        System.out.print("Enter the element to search: ");
        int search = input.nextInt();
        boolean found = false;
        for (int i = 0; i < size; i++) {
            if (arr[i] == search) {
                System.out.println(search + " found at position " + (i + 1));
                found = true;
                break;
            }
        }
        if (!found) {
            System.out.println(search + " not found in the array.");
        }
    }
}
